.agents/skills/aiq-add-data-source/references/package-layout.md

```
@@ -0,0 +1,98 @@
+<!--
+SPDX-FileCopyrightText: Copyright (c) 2026, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+SPDX-License-Identifier: Apache-2.0
+-->
+
+# Data source package layout
+
+Model new sources on `sources/google_scholar_paper_search/`. The layout is:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)